apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/get-editor-type.ts

```diff
@@ -0,0 +1,17 @@
+import { select } from '@wordpress/data';
+
+type EditorType = 'site' | 'post' | 'page';
+export const getEditorType = (): EditorType | undefined => {
+	const editorSelector = select( 'core/editor' );
+	const postType = editorSelector.getCurrentPostType();
```

Comment:
Since `getCurrentPostType` returns the post type, I assumed that if it is null, it is `site`.

Docs 📝
https://developer.wordpress.org/block-editor/reference-guides/data/data-core-editor/#getcurrentposttype
https://developer.wordpress.org/reference/classes/wp_post/

Reply:
Hi @okmttdhr, 

I am new to the codebase.

1. How do you know that `getCurrentPostType()` yields the value `site`. Looking in https://developer.wordpress.org/reference/classes/wp_post/ and navigating to `$post_type`, there is a comment that mentions `post` and `page`, but `site` is not mentioned in that section.
2.  Are there any more docs I should read to gain context on the values of `getCurrentPostType()`?

Reply:
Hi @and1can, thank you for your question.

> How do you know that getCurrentPostType() yields the value site

`getCurrentPostType` does **not** return `site` as a value since `getCurrentPostType` returns post type. 
Please take a look at this document. https://developer.wordpress.org/themes/basics/post-types/

With `getCurrentPostType`, we at least know if it is `post` or `page`. In this case, we wanted to distinguish `post`, `page`, and `site`, so I treated the `null` case as `site`.

Reply:
https://github.com/Automattic/wp-calypso/pull/73800#issuecomment-1449349064